README.md

```diff
@@ -3,7 +3,9 @@
 
 **Discord Server:** https://discord.gg/Buktv3t
 
-**Latest Builds for Windows, Linux (AppImage), and Android** https://github.com/stenzek/duckstation/releases/tag/latest
+**Latest Builds for Windows and Android** https://github.com/stenzek/duckstation/releases/tag/latest
+
+**Linux** https://github.com/stenzek/duckstation#linux
```

Comment:
I've created a link here to reference both the AppImage and Flatpak installation methods. The later is put forward because it is cross-distribution.